scipy/stats/_distn_infrastructure.py

```diff
@@ -3855,3 +3873,437 @@ def get_distribution_names(namespace_pairs, rv_base_class):
+            the negative log-likelihood function given ``x``, `dist`, and the
+            provided `data`.
+            The job of `optimizer` is to find values of the decision variables
+            that minimizes `fun`.
+        bounds : sequence of tuples
+            The bounds on values of the decision variables; each element will
+            be a tuple containing the lower and upper bound on a decision
+            variable.
+
+        If the distribution has any shape parameters that must be integral or
+        if the distribution is discrete and the location parameter is not
+        fixed, `optimizer` must also accept the following keyword argument.
+
+        integrality : array_like of bools
+            For each decision variable, True if the decision variable is
+            must be constrained to integer values and False if the decision
+            variable is continuous.
+
+        `optimizer` must return an object, such as an instance of
+        `scipy.optimize.OptimizeResult`, which holds the optimal values of
+        the decision variables in an attribute ``x``. If attributes
+        ``fun``, ``status``, or ``message`` are provided, they will be
+        included in the result object returned by `fit`.
+
+    Returns
+    -------
+    result : FitResult
+        An object with the following fields.
+
+        dist : scipy.stats.rv_continuous or scipy.stats.rv_discrete
+            The distribution object passed to `fit` as `dist`
+        data : 1D array_like
+            The data passed to `fit` as `data`
+        nllf : callable
+            The negative log-likehood function for the given `data`. Accepts a
+            single tuple containing the shapes, location, and scale of the
+            distribution.
+        params : namedtuple
+            A namedtuple containing the maximum likelihood estimates of the
+            shape parameters, location, and (if applicable) scale of the
+            distribution.
+        objective_val : float
+            The value of the negative log likelihood function evaluated with
+            the fitted shapes, i.e. ``result.nllf(result.params)``.
+        success : bool or None
+            Whether the optimizer considered the optimization to terminate
+            successfully or not.
+        message : str or None
+            Any status message provided by the optimizer.
+
+    See Also
+    --------
+    rv_continuous,  rv_discrete
+
+    Notes
+    -----
+    Optimization is more likely to converge to the maximum likelihood estimate
+    when the user provides tight bounds containing the maximum likelihood
+    estimate. For example, when fitting a binomial distribution to data, the
+    number of experiments underlying each sample may be known, in which case
+    the corresponding shape parameter ``n`` can be fixed.
+
+    Examples
+    --------
+    Suppose we wish to fit a distribution to the following data.
+
+    >>> import numpy as np
```

Comment:
Until we conclude on the other issue we should remove this...
```suggestion
```

Reply:
I'm not sure what to say. I don't want to start a fuss, but especially since the discussion is active elsewhere and not decided, I don't think it should be debated here either way.